test/src/test/java/org/corfudb/runtime/collections/CorfuStoreShimTest.java

```diff
@@ -1054,28 +1056,34 @@ public void checkProtoDeduplication() throws Exception {
             assertThat(referenceMap.get(fileName.getFileName())).isNotNull();
         }
 
-        // Now update the schemas with different protobuf files
-        final int numProtoFiles = corfuRuntime.getTableRegistry().getProtobufDescriptorTable().size();
-        assertThat(numProtoFiles).isLessThan(numTables);
-        for (int i = 0; i < numTables; i++) {
-            shimStore.openTable(
-                    someNamespace,
-                    tableNamePrefix+i,
-                    UuidMsg.class,
-                    org.corfudb.runtime.proto.LogData.LogDataMsg.class, // this brings in 1 new protobuf file
-                    ManagedMetadata.class,
-                    // TableOptions includes option to choose - Memory/Disk based corfu table.
-                    TableOptions.builder().build());
-        }
+        final int numProtoFiles =
+                corfuRuntime.getTableRegistry().getProtobufDescriptorTable().size();
+        CorfuStoreMetadata.TableName tableName = CorfuStoreMetadata.TableName
+                .newBuilder()
+                .setNamespace(someNamespace)
+                .setTableName(tableNamePrefix+"0")
+                .build();
+        final Collection<CorfuRecord<ProtobufFileDescriptor, CorfuStoreMetadata.TableMetadata>> records =
+                corfuRuntime.getTableRegistry().getProtobufDescriptorTable().values();
+
+        shimStore.openTable(
```

Comment:
nit - consider adding a comment here saying that we open one of the tables with a different schema and validate no checks for schema update are made.